.github/pull_request_template.md

```diff
@@ -0,0 +1,3 @@
+:warning: Only merge changes if you are happy for them to go live. :warning: 
```

Comment:
```suggestion
## What
<!-- Description of the change being made -->

## Why
<!-- What are the reasons behind this change being made? -->

## Visual Changes
<!-- If the change results in visual changes, show a before and after -->


:warning: Only merge changes if you are happy for them to go live. :warning: 
```

Reply:
So I had originally added something like this, but Ben pointed out that it [maybe out of scope for this PR](https://github.com/alphagov/smart-answers/pull/4606#discussion_r444783642).

Seems like people may have differing views about PR templatse and I think it needs a bit more discussion, so I'm going to leave it for a future PR. For now it'd be good to have at least the warning about continuous delivery.